interfaces/portal/src/app/services/registration-attribute.service.ts

```diff
@@ -184,6 +200,13 @@ export class RegistrationAttributeService {
         options,
         value,
         type,
+        isEditable: this.isEditableAttribute({
+          attributeName,
+          project,
+        }),
+        isRequired:
+          attributeName === GenericRegistrationAttributes.phoneNumber &&
+          !project.allowEmptyPhoneNumber,
```

Comment:
Maybe an:
```suggestion
        isRequired: this.isRequiredAttribute({
          attributeName: GenericRegistrationAttributes.phoneNumber,
          project,
        }),
```

To get the same shape as "isEditable"?

Reply:
As soon there is a scenario of some other specific attribute that needs some `isRequired`-check; _Then_ we can solve it. ;)